internal/environment/command_create.go

```diff
@@ -21,7 +21,7 @@ func (c *command) newCreateCommand() *cobra.Command {
 		RunE:  c.create,
 	}
 
-	c.addStreamGovernancePackageFlag(cmd, "essentials")
+	c.addStreamGovernancePackageFlag(cmd, "")
```

Comment:
This is a breaking change... unless the backend treats `""` as `"essentials"`?

Reply:
(No need to wait for v4 if the above is true)

Reply:
I think the idea of this PR is that introducing this itself was a breaking change, so reversing that should just be a bug fix, right?

Reply:
Wait, we accidentally merged a breaking change and this reverts it?

Reply:
Correct. The changes were made in https://github.com/confluentinc/cli/pull/2409. The "breaking" part of the changes are explained [here](https://confluentinc.atlassian.net/wiki/x/twI3xQ)